csrankings-d.csv

```diff
@@ -1276,6 +1276,7 @@ Dong Seong Kim,University of Canterbury,http://cosc.canterbury.ac.nz/dongseong.k
 Dong Shao,Nanjing University,https://software.nju.edu.cn/dongshao/index.html,NOSCHOLARPAGE
 Dong Soo Han,KAIST,https://cs.kaist.ac.kr/people/view?idx=369&kind=faculty&menu=160,nxoFaYAAAAAJ
 Dong Tong 0001,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6017,NOSCHOLARPAGE
+Dong Wang 0002,Univ. of Illinois at Urbana-Champaign,https://www.wangdong.org,-NfMhb0AAAAJ

```

Comment:
Not in CS. Please re-open with evidence to justify how this faculty member meets the inclusion criteria (can solely advise a PhD student for a PhD in *Computer Science*). Thanks.